src/Entity/Workout.php

```diff
@@ -75,20 +87,7 @@ public function setName(string $name): self
 
     public function getDuration(): ?int
     {
-        return count($this->exercises);
-    }
-
-
-    public function getExercises(): ?array
-    {
-        return $this->exercises;
-    }
```

Comment:
@daria-sieroshtan 
I think you are going to need a method `gtExcercises()` in this entity.
It should go through `workoutExcercise`'s, order them, fetch exercises for each